docs/how-to-guides/data-basics/5-adding-a-delete-button.md

```diff
@@ -0,0 +1,454 @@
+# Adding a delete button
+
+In this section, we will add a *Delete* feature to our app. Here's a glimpse of what we're going to build:
+
+![](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/how-to-guides/data-basics/media/delete-button/delete-button.png)
+
+### Step 1: Add a _Delete_ button
+
+Let's start by creating the `DeletePageButton` component and updating the user interface of our `PagesList` component:
+
+```js
+import { Button } from '@wordpress/components';
+import { decodeEntities } from '@wordpress/html-entities';
+
+const DeletePageButton = () => (
+	<Button variant="primary">
+		Delete
+	</Button>
+)
+
+function PagesList( { hasResolved, pages } ) {
+	if ( ! hasResolved ) {
+		return <Spinner />;
+	}
+	if ( ! pages?.length ) {
+		return <div>No results</div>;
+	}
+
+	return (
+		<table className="wp-list-table widefat fixed striped table-view-list">
+			<thead>
+				<tr>
+					<td>Title</td>
+					<td style={{width: 190}}>Actions</td>
+				</tr>
+			</thead>
+			<tbody>
+				{ pages?.map( ( page ) => (
+					<tr key={page.id}>
+						<td>{ decodeEntities( page.title.rendered ) }</td>
+						<td>
+							<div className="form-buttons">
+								<EditPageButton pageId={ page.id } />
+								<DeletePageButton pageId={ page.id } />
```

Comment:
It would be cool if we had a way to highlight the addition somehow.

Reply:
I know, right? Maybe I could add a comment like 

``` This is the addition v```

Or! Use the `diff` format 

```diff
	<div className="form-buttons">
								<EditPageButton pageId={ page.id } />
+								<DeletePageButton pageId={ page.id } />
```

Reply:
I went for
```
{/* ↓ This is the only change in the PagesList component */}
<DeletePageButton pageId={ page.id }/>
{/* ↑ This is the only change in the PagesList component */}
```